src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -319,7 +319,6 @@ namespace AppInstaller::CLI::Workflow
             ScanArchiveFromLocalManifest <<
             ExtractFilesFromArchive <<
             VerifyAndSetNestedInstaller <<
-            GetInstallerArgs <<

```

Comment:
GetInstallerArgs is removed because it will be invoked accordingly in the ExecuteInstallerForType below.